windows-installer/generate_sdl_agent_exe.ps1

```diff
@@ -135,18 +139,18 @@ $env:Path = [System.Environment]::GetEnvironmentVariable("Path","Machine") + ";"
 #  STEP 4 - INSTALL THE GEMS
 ##############################
 #
-# Install the needed gems for google fleuntd to works.
+# Install the needed gems for google fluentd to work.
 #
 # These are all a known set of versions which work together. Be sure things work if you
 # update any of them.
-#
-# We install all gems with '--no-document' so we don't pull in
-# unneeded docs that bloat the file size (and also seem to cause issues with unzipping).
 ###############################
 
+$gem_installer = $SRC_ROOT + '\bin\gem_installer'
+$core_gems_rb = $SRC_ROOT + '\core_gems.rb'
+$plugin_gems_rb = $SRC_ROOT + '\plugin_gems.rb'
 & $GEM_CMD install fluentd:1.4.2 --no-document
-& $GEM_CMD install windows-pr:1.2.6 win32-ipc:0.7.0 win32-event:0.6.3 win32-eventlog:0.6.7 win32-service:2.1.4 fluent-plugin-windows-eventlog:0.2.2 --no-document
-& $GEM_CMD install google-protobuf:3.7.1 grpc:1.20.0 fluent-plugin-google-cloud:0.7.23 --no-document
+& $RUBY_EXE $gem_installer $core_gems_rb
```

Comment:
Now that there are no Windows-specific core_gems changes, we probably don't need this anymore.

Reply:
I'll leave it this way for consistency. We'll build something closer to what we build on Linux, maybe... and if anything breaks specifically on Windows, we'll know.